lib/concurrent/executor/java_executor_service.rb

```diff
@@ -18,10 +18,6 @@ class JavaExecutorService < AbstractExecutorService
       }.freeze
       private_constant :FALLBACK_POLICY_CLASSES
 
-      def initialize(*args, &block)
```

Comment:
@pitr-ch I'm not sure, maybe it is required to have it declared in a java-based versions?